src/test/regress/upgrade/custom_citus.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+
+"""custom_citus
+Usage:
+    custom_citus --bindir=<bindir> --pgxsdir=<pgxsdir> --parallel=<parallel>
+
+Options:
+    --bindir=<bindir>              The PostgreSQL executable directory(ex: '~/.pgenv/pgsql-11.3/bin')
+    --pgxsdir=<pgxsdir>           	       Path to the PGXS directory(ex: ~/.pgenv/src/postgresql-11.3)
+    --parallel=<parallel>           how many configs to run in parallel
+"""
+
+import upgrade_common as common
+import threading
+import atexit
+from docopt import docopt
+import os, shutil
+import time
+import sys
+
+from config import (
```

Comment:
maybe use `import config as cfg` and then use `cfg.CitusDefaultClusterConfig` (etc) in the code. Most of these variables are only used once, so it doesn't help much to pull them all into scope.